[Quarkus 3.0.0.Alpha6] IllegalAnnotationsException Two classes have the same XML type name thrown at build time by quarkus-jaxb

Something must have changed in Quarkus JAXB because all of the CXF SOAP tests fail with similar `Two classes have the same XML type` issues like this:

```
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
        [error]: Build step io.quarkus.jaxb.deployment.JaxbProcessor#setupJaxbContextConfig threw an exception: java.lang.IllegalStateException: Failed to configure JAXB context
        at io.quarkus.jaxb.deployment.JaxbProcessor.validateContext(JaxbProcessor.java:392)
        at io.quarkus.jaxb.deployment.JaxbProcessor.setupJaxbContextConfig(JaxbProcessor.java:317)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
        at io.quarkus.builder.BuildContext.run(BuildContext.java:282)
        at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
        at java.base/java.lang.Thread.run(Thread.java:833)
        at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: org.glassfish.jaxb.runtime.v2.runtime.IllegalAnnotationsException: 47 counts of IllegalAnnotationExceptions
Two classes have the same XML type name "{http://www.w3.org/2000/09/xmldsig#}ObjectType". Use @XmlType.name and @XmlType.namespace to assign different names to them.
        this problem is related to the following location:
                at org.apache.cxf.ws.security.sts.provider.model.xmldsig.ObjectType
        this problem is related to the following location:
                at org.apache.xml.security.binding.xmldsig.ObjectType
                at public jakarta.xml.bind.JAXBElement org.apache.xml.security.binding.xmldsig.ObjectFactory.createObject(org.apache.xml.security.binding.xmldsig.ObjectType)
                at org.apache.xml.security.binding.xmldsig.ObjectFactory
                at protected java.util.List org.apache.xml.security.binding.xmldsig.TransformType.content
                at org.apache.xml.security.binding.xmldsig.TransformType
                at protected java.util.List org.apache.xml.security.binding.xmlenc.TransformsType.transform
                at org.apache.xml.security.binding.xmlenc.TransformsType
                at public org.apache.xml.security.binding.xmlenc.TransformsType org.apache.xml.security.binding.xmlenc.ObjectFactory.createTransformsType()
                at org.apache.xml.security.binding.xmlenc.ObjectFactory

```

--- a/integration-test-groups/cxf-soap/cxf-soap-client/src/main/resources/application.properties
+++ b/integration-test-groups/cxf-soap/cxf-soap-client/src/main/resources/application.properties
@@ -17,11 +17,6 @@
 
 quarkus.cxf.path=/soapservice
 quarkus.native.resources.includes = wsdl/*.wsdl
-
-# Workaround for https://github.com/apache/camel-quarkus/issues/4603
-# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
-quarkus.jaxb.validate-jaxb-context = false
-
 # disable the root config
 quarkus.cxf.codegen.wsdl2java.includes = none.wsdl
 # ... and do everything with named parameter sets so that it works in the grouped module

--- a/integration-test-groups/cxf-soap/cxf-soap-mtom-awt/src/main/resources/application.properties
+++ b/integration-test-groups/cxf-soap/cxf-soap-mtom-awt/src/main/resources/application.properties
@@ -17,7 +17,3 @@
 
 quarkus.cxf.path=/soapservice
 quarkus.native.resources.includes = wsdl/*.wsdl
-
-# Workaround for https://github.com/apache/camel-quarkus/issues/4603
-# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
-quarkus.jaxb.validate-jaxb-context = false

--- a/integration-test-groups/cxf-soap/cxf-soap-mtom/src/main/resources/application.properties
+++ b/integration-test-groups/cxf-soap/cxf-soap-mtom/src/main/resources/application.properties
@@ -17,7 +17,3 @@
 
 quarkus.cxf.path=/soapservice
 quarkus.native.resources.includes = wsdl/*.wsdl
-
-# Workaround for https://github.com/apache/camel-quarkus/issues/4603
-# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
-quarkus.jaxb.validate-jaxb-context = false

--- a/integration-test-groups/cxf-soap/cxf-soap-server/src/main/resources/application.properties
+++ b/integration-test-groups/cxf-soap/cxf-soap-server/src/main/resources/application.properties
@@ -18,10 +18,6 @@
 quarkus.cxf.path=/soapservice
 quarkus.native.resources.includes = wsdl/*.wsdl
 
-# Workaround for https://github.com/apache/camel-quarkus/issues/4603
-# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
-quarkus.jaxb.validate-jaxb-context = false
-
 # disable the root config
 quarkus.cxf.codegen.wsdl2java.includes = none.wsdl
 # ... and do everything with named parameter sets so that it works in the grouped module

--- a/integration-test-groups/cxf-soap/cxf-soap-ws-security-client/src/main/resources/application.properties
+++ b/integration-test-groups/cxf-soap/cxf-soap-ws-security-client/src/main/resources/application.properties
@@ -18,10 +18,6 @@
 quarkus.cxf.path=/soapservice
 quarkus.native.resources.includes = wsdl/*.wsdl
 
-# Workaround for https://github.com/apache/camel-quarkus/issues/4603
-# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
-quarkus.jaxb.validate-jaxb-context = false
-
 # disable the root config
 quarkus.cxf.codegen.wsdl2java.includes = none.wsdl
 # ... and do everything with named parameter sets so that it works in the grouped module

--- a/integration-test-groups/cxf-soap/cxf-soap-ws-security-server/src/main/resources/application.properties
+++ b/integration-test-groups/cxf-soap/cxf-soap-ws-security-server/src/main/resources/application.properties
@@ -20,7 +20,3 @@ quarkus.cxf.path=/soapservice
 quarkus.cxf.endpoint."/security-policy-hello-cxf-way".implementor=org.apache.camel.quarkus.component.cxf.soap.securitypolicy.server.cxf.way.it.WssSecurityPolicyHelloServiceCxfWayImpl
 
 quarkus.native.resources.includes=bob.properties,alice.properties,alice.jks,bob.jks,encrypt-sign-policy.xml
-
-# Workaround for https://github.com/apache/camel-quarkus/issues/4603
-# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
-quarkus.jaxb.validate-jaxb-context = false

--- a/integration-test-groups/cxf-soap/cxf-soap-ws-trust/src/main/resources/application.properties
+++ b/integration-test-groups/cxf-soap/cxf-soap-ws-trust/src/main/resources/application.properties
@@ -22,7 +22,3 @@ quarkus.cxf.endpoint."/jaxws-samples-wsse-policy-trust-sts".features=org.apache.
 quarkus.cxf.endpoint."/jaxws-samples-wsse-policy-trust-cxf-way".implementor=org.apache.camel.quarkus.component.cxf.soap.it.ws.trust.server.cxf.way.TrustHelloServiceCxfWayImpl
 
 quarkus.native.resources.includes=*.properties,*.jks,*.wsdl,*.xml,*.xsd
-
-# Workaround for https://github.com/apache/camel-quarkus/issues/4603
-# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
-quarkus.jaxb.validate-jaxb-context = false

